site/data/i18n/en.json

```diff
@@ -30,7 +30,7 @@
 	"menu-main-benefits-link-main": "https://www.canada.ca/en/services/benefits.html",
 	"menu-main-benefits-id-main": "benefits",
 	"menu-main-health-text-main": "Health",
-	"menu-main-health-link-main": "https://www.canada.ca/en/services/health.html",
+	"menu-main-health-link-main": "http://healthycanadians.gc.ca/index-eng.php",
```

Comment:
Sorry :(. Looks like I messed up my rebase/conflict resolutions.

I was under the mistaken impression that the only changes that had been made to these files since this PR was opened had come from #1213. So while rebasing, I re-used my versions of those files as-is without any second thought.

I'll sort it out soon.